includes/multi-currency/Helpers/OrderMetaHelper.php

```diff
@@ -39,7 +39,18 @@ class OrderMetaHelper {
 	 */
 	public function __construct( WC_Payments_API_Client $payments_api_client ) {
 		$this->payments_api_client = $payments_api_client;
-		$this->add_actions();
+	}
+
+	/**
+	 * Initializes this class' WP hooks.
+	 *
+	 * @return void
+	 */
+	public function init_hooks() {
```

Comment:
This class already had a separate method to load hooks/filters, so just renamed it and made it public so it could be called externally outside of the constructor.